### ISSUE TITLE
maskrc-svg v0.5-1 does not work with gubbins v2.3.4 or v2.4.1

### ISSUE THREAD

**Issue by eam12** [kwongj/maskrc-svg#8]
I am installing both `Gubbins` and `maskrc-svg` using conda:

- [Gubbins](https://anaconda.org/bioconda/gubbins)
- [maskrc-svg](https://anaconda.org/bioconda/maskrc-svg)

In the past, I have successfully used `maskrc-svg v0.5` with both `Gubbins v2.3.4` and `v2.4.1`. However, with the recent bioconda switch to `maskrc-svg v0.5-1` I get the following error when I run with `Gubbins`:

```
% maskrc-svg.py --aln clean.full.aln --gubbins gubbins --symbol N --out clean.full.noRecomb.aln --svg core_recombinant_regions.svg
Traceback (most recent call last):
  File "/home/user/.conda/envs/maskrc-svg2/bin/maskrc-svg.py", line 36, in <module>
    from ete3 import Tree
ModuleNotFoundError: No module named 'ete3'
```

I did confirm dependency `ete3` was loaded and working (v3.1.1):  

```
% ete3 -h
usage: ete3 [-h]
            {mod,extract,annotate,compare,view,ncbiquery,generate,evol,maptrees,build,version,upgrade-external-tools}
            ...

optional arguments:
  -h, --help            show this help message and exit

AVAILABLE PROGRAMS:
  {mod,extract,annotate,compare,view,ncbiquery,generate,evol,maptrees,build,version,upgrade-external-tools}
```

Any help would be much appreciated! Thank you!

**Comment by evezeyl** [comment]
Hi had the same error, when installed directly from anaconda cloud . (https://anaconda.org/bioconda/maskrc-svg). So installed first a conda python3 environment< [and then tried with pip and github install (somehow did not work properly)]. So I tried to install anyway maskrc-svg within the python3 environment, and this seems to be working now.
Home this might help for the solution.
All the best
Eve

**Comment by eam12** [comment]
Hi Eve,
I'm not sure I totally understand what you did. Do you mean that you installed the anaconda cloud version of maskrc-svg and then added the github version of maskrc-svg?
Thank you for your help!
Liz

**Comment by evezeyl** [comment]
I an anaconda environment with python3 - and then within this environment I actually installed the conda version of maskrc-svg. (I tried with the pip for github but that did not work, so I took chance and installed the conda version).I suspect there is a version problem with ete3 dependencies. 
so you can try like that:
```
conda create -n maskrc-svg python=3
# when this is done do: 
conda activate maskrc-svg
conda install -c bioconda maskrc-svg
```
then If its like me, it might work. Good luck. Eve

**Comment by eam12** [comment]
Hi Eve,
Unfortunately, that is _exactly_ what I'm already doing. I have no idea what the problem could be! I really appreciate your help.
Thank you!

**Comment by evezeyl** [comment]
I would be you I would delete my environment and start from scratch...and retry
And if this does not work you can try with pip 
ok, then you can try to install it from github with pip 
In your activated conda maskrc-svg with python3 you can try: 
```
conda install git pip
pip install git+git://github.com/kwongj/maskrc-svg@master
```
And if this still does not work, well you will have to wait that the author help you, or to find a person around you that can help troubleshoot. I only posted the solution that worked for me so ...

Good luck
Eve

**Comment by eam12** [comment]
Looks like the issue was with my version of python3. The newest version of Gubbins available via conda requires python3/3.7.4, but I was running python3/3.6.3.
